.github/workflows/php-lint-tests.yml

```diff
@@ -105,6 +105,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install SVN
+        run: sudo apt-get update && sudo apt-get install -y subversion
```

Comment:
Another option we have is to clone the WordPress Git repo rather than exporting the canonical SVN one: https://make.wordpress.org/core/handbook/contribute/git/

Let's stick with SVN for now, but we might want to consider the above for a future iteration. We could trim a minute or so off each PHP test run in CI by not installing the SVN package, although we'd need to weigh up the pros and cons of using the Git mirror over the canonical SVN repository. It would also be interesting to see the relative performance of the Git vs SVN checkout.